Proposals/xgov-196.md

```diff
@@ -0,0 +1,107 @@
+---
+id: 196
+period: 4
+title: TealCraft - Remix-like IDE for TealScript
+author: Shivaprasad Manupadi (@shivamanupadi)
+email: shivaprasadmanupadi@gmail.com
+discussions-to: https://forum.algorand.org/t/xgov-196-tealcraft-remix-like-ide-for-tealscript/11845
+company_name: TealCraft
+category: Tools
+focus_area: IDE
+open_source: Yes
+funding_type: Retroactive
+amount_requested: 195000
+delivery_date: 2024-04-20
+status: Final
+---
+
+## Abstract
+Remix-like web IDE to write, compile, test deploy, and invoke smart contract calls on Algorand using TealScript.
+
+## Team
+Shivaprasad - Developer
+- 10 years of experience in building web applications.
+- Core expertise in TypeScript.
+
+## Experience with Algorand
+
+Developer of AlgoDesk, DappFlow, and PixelNode, contributing to the Algorand ecosystem over the past 2 years.
+
+## Present Proposal
+
```

Comment:
Please add the subpart with relevant information about the deliverable here.

```suggestion
### Deliverable
Proposals must include clear, non-technical descriptions of deliverables.
```

Reply:
> Please add the subpart with relevant information about the deliverable here.

Done